content/en/getting_started/integrations/google_cloud.md

```diff
@@ -0,0 +1,393 @@
+---
+title: Getting Started with Google Cloud
+further_reading:
+    - link: 'https://docs.datadoghq.com/integrations/google_cloud_platform/?tab=dataflowmethodrecommended'
+      tag: 'Documentation'
+      text: 'Google Cloud integration'
+    - link: 'https://docs.datadoghq.com/account_management/billing/google_cloud/'
+      tag: 'Guide'
+      text: 'Google Cloud integration billing'
+    - link: 'https://docs.datadoghq.com/integrations/guide/cloud-metric-delay/'
+      tag: 'Guide'
+      text: 'Cloud Metric Delay'
+    - link: 'https://docs.datadoghq.com/agent/guide/why-should-i-install-the-agent-on-my-cloud-instances/'
+      tag: 'Guide'
+      text: 'Why should I install the Datadog Agent on my cloud instances?'
+    - link: 'https://www.datadoghq.com/blog/gke-dashboards-integration-improvements/'
+      tag: 'Blog'
+      text: 'New GKE dashboards and metrics provide deeper visibility into your environment'
+    - link: 'https://www.datadoghq.com/blog/google-cloud-private-service-connect/'
+      tag: 'Blog'
+      text: 'Access Datadog privately and monitor your Google Cloud Private Service Connect usage'
+    - link: 'https://www.datadoghq.com/blog/track-bigquery-costs-performance/'
+      tag: 'Blog'
+      text: 'Monitor BigQuery with Datadog'
+    - link: 'https://www.datadoghq.com/blog/google-cloud-cost-management/'
+      tag: 'Blog'
+      text: 'Empower engineers to take ownership of Google Cloud costs with Datadog'
+    - link: 'https://www.datadoghq.com/blog/collect-traces-logs-from-cloud-run-with-datadog/'
+      text: 'Collect traces, logs, and custom metrics from your Google Cloud Run services with Datadog'
+      tag: 'Blog'
+---
+
+## Overview
+
+Integrating your Google Cloud projects with Datadog allows you to monitor your GCE and GKE compute instances within the Datadog platform.
```

Comment:
Given that most of the related integrations expand beyond GCE and GKE we should probably have that reflected in this main overview sentence
```suggestion
Integrating your Google Cloud projects with Datadog allows you to monitor your Google Cloud services within the Datadog platform.
```